hrm-domain/hrm-core/contact/adminContactRoutesV0.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import type { Request, Response, NextFunction } from 'express';
+import { isErr, mapHTTPError } from '@tech-matters/types';
+// import createError from 'http-errors';
+import { SafeRouter, publicEndpoint } from '../permissions';
+import { reindexContacts } from './contactsReindexService';
+
+const adminContactsRouter = SafeRouter();
+
+// admin POST endpoint to reindex contacts. req body has accountSid, dateFrom, dateTo
+adminContactsRouter.post(
```

Comment:
This endpoint only allows us to reindex one helpline at a time. I can see needing to reindex all helplines in a DB to be quite a common use case. Are we going to implement that at the script level, make a separate call to this API per helpline? @GPaoloni